package.go

```diff
@@ -17,6 +17,8 @@ type Package struct {
 	Source *Package `json:"source,omitempty"`
 	// the file system path or prefix where this package resides
 	PackageDB string `json:"-"`
+	// a location in the layer where the package is located, this is useful for language packages.
```

Comment:
what is the difference between this and `PackageDB`?

Reply:
`PackageDB` is more used for storing the location of the DB/metadata (bdb, alpine `installed`, etc.), it's deletion doesn't mean that the packages are deleted, it just provides some context.